pychord/constants/qualities.py

```diff
@@ -9,6 +9,7 @@
     # 3 notes
     ('', (0, 4, 7)),
     ('maj', (0, 4, 7)),
+    ('-', (0, 3, 7)),
```

Comment:
Could you swap this line with the next? The more common notations are listed above in this OrderedDict.

Reply:
Yes!